app/views/templates/playerTimeline.jst

```diff
@@ -1,5 +1,7 @@
 <div class="text-container">
-    <p id="trackname"><%= title %></p>
+    <p id="title"><%= title %></p>
+    <p id="separator">&nbsp;-&nbsp;</p>
```

Comment:
You can make a clever use of markup and CSS here:

1. prefer to use `<span>` instead of `<p>` for inline contents
2. the separator here is only cosmetic, consider a CSS `#artist::before` as a better option ;)